### PR TITLE
Added support for GCP connection

### PR DESCRIPTION
**Pull request by om-shrirao**

### CODE REVIEW

File: internal/provider/connections/resource_gcp_connection.go [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+	plan.UpdatedAt = types.StringValue(gjson.Get(response, "updatedAt").String())
+	plan.CreatedAt = types.StringValue(gjson.Get(response, "createdAt").String())
+	plan.Category = types.StringValue(gjson.Get(response, "category").String())
+	plan.Service = types.StringValue(gjson.Get(response, "service").String())
+	plan.ResourceURL = types.StringValue(gjson.Get(response, "resource_url").String())
+	plan.LastConnectionOK = types.BoolValue(gjson.Get(response, "last_connection_ok").Bool())
+	plan.LastConnectionError = types.StringValue(gjson.Get(response, "last_connection_error").String())
+	plan.LastConnectionAt = types.StringValue(gjson.Get(response, "last_connection_at").String())
+
+	tflog.Debug(ctx, fmt.Sprintf("Response: %s", response))
```
Comment (VJ-Vijay77) [inline internal/provider/connections/resource_gcp_connection.go:180]: we can remove this debug log, below is the same log printing the response

Comment (VJ-VijayThales) [inline internal/provider/connections/resource_gcp_connection.go:180]: we can remove this debug log as the below log is also the same printing the response

File: internal/provider/connections/resource_gcp_connection.go [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+
+	tflog.Debug(ctx, "[resource_gcp_connection.go -> Create Output]["+response+"]")
```
Comment (VJ-VijayThales) [inline internal/provider/connections/resource_gcp_connection.go:182]: we can remove this debug log as the below log is also the same printing the response

File: internal/provider/connections/resource_gcp_connection.go (hunk 1 above)
Comment (VJ-VijayThales) [inline internal/provider/connections/resource_gcp_connection.go:180]: we can remove this debug log as the below log is also the same printing the response

File: internal/provider/resource_scp_connection_test.go [hunk 3]
```diff
@@ -10,7 +10,7 @@ func TestResourceCMSCPConnection(t *testing.T) {
 	resource.Test(t, resource.TestCase{
 		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
 		Steps: []resource.TestStep{
-			// creating a SCP connection
```
Comment (VJ-VijayThales) [inline internal/provider/resource_scp_connection_test.go:13]: I havent given this much priority to grammar :)))

**Review by VJ-VijayThales — APPROVED** [review]

**Pull request merged**